src/main/java/com/stripe/net/UntypedMapDeserializer.java

```diff
@@ -0,0 +1,71 @@
+package com.stripe.net;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+
+public class UntypedMapDeserializer {
```

Comment:
Old logic ported from `EventDataDeserializer`. New tests are added to `UntypedMapDeserializerTest`

Reply:
And similarly here, can we do a docblock?